Source/Core/DiscIO/VolumeWiiCrypted.cpp

```diff
@@ -160,10 +160,43 @@ IVolume::ECountry CVolumeWiiCrypted::GetCountry() const
 	if (!m_pReader)
 		return COUNTRY_UNKNOWN;
 
+	u32 region_code;
+	m_pReader->Read(0x4E000, sizeof(u32), (u8*)&region_code);
+	region_code = Common::swap32(region_code);
```

Comment:
You can use `ReadSwapped(0x4E000, &region_code, false)` instead of `m_pReader->Read`. This will get rid of the cast and the swap line.